frontend/packages/dev-console/src/components/taskruns/TaskRunLog.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { useK8sGet } from '@console/internal/components/utils/k8s-get-hook';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { PodLogs } from '@console/internal/components/pod-logs';
+import { StatusBox } from '@console/internal/components/utils/status-box';
+import { PodModel } from '@console/internal/models';
+
+export type TaskRunLogProps = {
+  obj: K8sResourceKind;
+};
+
+const TaskRunLog: React.FC<TaskRunLogProps> = ({ obj }) => {
+  const [pod, podLoaded, podError] = useK8sGet<K8sResourceKind>(
+    PodModel,
+    obj.status.podName,
+    obj.metadata.namespace,
+  );
+  if (podLoaded) {
```

Comment:
```suggestion
  if (podLoaded && !podError) {
```

Reply:
updated this as well as named to `TaskRunLogWrapper`